admin-part/src/main/kotlin/Coverage.kt

```diff
@@ -53,13 +53,14 @@ internal fun Sequence<Session>.calculateCoverageData(
     logger.info { coverageByType }
 
     val methodCount = bundleCoverage.methodCount.copy(total = context.packageTree.totalMethodCount)
+    val buildMethods = context.calculateBundleMethods(bundleCoverage)
     val coverageBlock: Coverage = when (scope) {
         null -> {
             BuildCoverage(
                 ratio = totalCoveragePercent,
                 count = coverageCount,
                 methodCount = methodCount,
-                riskCount = zeroCount,
+                riskCount = buildMethods.risksCount(),
```

Comment:
risks should be added to the coverage before sending 
method calculateAndSendBuildCoverage (Plugin.kt)